c++/src/capnp/rpc-prelude.h

```diff
@@ -106,6 +106,7 @@ class RpcSystemBase {
   ~RpcSystemBase() noexcept(false);
 
   void setTraceEncoder(kj::Function<kj::String(const kj::Exception&)> func);
+  int countQuestionsForTest();
```

Comment:
Feels like a weird wart. Maybe we could have a `getMetrics()` method that returns a struct with sizes of all the tables? That's actually a reasonable public interface (for monitoring purposes).